amd/src/annoto.js

```diff
@@ -95,6 +95,7 @@ define([
                 vimeo = $(parent).find('iframe[src*="vimeo.com"]').first().get(0),
                 videojs = $(parent).find('.video-js').first().get(0),
                 jwplayer = $(parent).find('.jwplayer').first().get(0),
+                wistia = $(parent).find('.wistia_embed_initialized').first().get(0),
```

Comment:
@ilyaDymshits why `. wistia_embed_initialized` was used instead of `.wistia_embed`?